server/src/com/cloud/servlet/ConsoleProxyServlet.java

```diff
@@ -414,7 +423,7 @@ private String composeThumbnailUrl(String rootUrl, VirtualMachine vm, HostVO hos
         return sb.toString();
     }
 
-    private String composeConsoleAccessUrl(String rootUrl, VirtualMachine vm, HostVO hostVo) {
+    private String composeConsoleAccessUrl(String rootUrl, VirtualMachine vm, HostVO hostVo, Boolean webSocketRequest) {
```

Comment:
Same comment as above, the `Boolean` can be refactored into a native `boolean` and avoid unboxing and NPEs